bessctl/conf/samples/exactmatch.bess

```diff
@@ -33,12 +33,12 @@ em:1 -> Sink()
 em:2 -> Sink()
 em:5 -> Sink()   # used as default gate
 
-em.add(fields=[chr(17), aton('172.16.100.1'), aton('10.0.0.1')], gate=0)
-em.add(fields=[chr(6), aton('172.12.55.99'), aton('12.34.56.78')], gate=1)
-em.add(fields=[chr(17), aton('172.12.55.99'), aton('12.34.56.78')], gate=2)
+em.add(fields=[{'value_int':17}, {'value_bin':aton('172.16.100.1')}, {'value_bin':aton('10.0.0.1')}], gate=0)
```

Comment:
It would be nice if we can directly specify a scalar value for `oneof` variables, as long as there is no type ambiguity. For example, this line can be rewritten as `em.add(fields=[17, aton('172.16.100.1'), aton('10.0.0.1')], gate=0)`, which is shorter and sweeter.

It would require some non trivial changes in `protobuf_to_dict.py`...